src/api/environment.cc

```diff
@@ -543,7 +543,9 @@ void InitializeContextRuntime(Local<Context> context) {
   if (context->Global()->Get(context, intl_string).ToLocal(&intl_v) &&
       intl_v->IsObject()) {
```

Comment:
Ideally, we’d also return `Nothing<bool>()` if the `.ToLocal()` call here returns false (and ditto below)

Reply:
@addaleax done, PTAL